platzky/engine.py

```
@@ -3,24 +3,77 @@
+        return False
+
+    normalized_path = os.path.normpath(locale_dir)
+    if ".." in normalized_path.split(os.sep):
+        logger.warning("Rejected locale path with .. components: %s", locale_dir)
+        return False
+
+    locale_path = os.path.realpath(locale_dir)
+    module_path = os.path.realpath(os.path.dirname(module.__file__))
+
+    if not locale_path.startswith(module_path + os.sep):
+        if locale_path != module_path:
```

Ruff (0.15.12)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)